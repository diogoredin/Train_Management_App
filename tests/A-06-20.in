6
3
1
A1
B6
2017-12-23
05:00
0
0
0
