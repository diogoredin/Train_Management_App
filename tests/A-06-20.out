Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
Itinerário 1 para 2017-12-23 @ 8.24
Serviço #590 @ 0.36
07:02 A1
07:03 A2
07:04 B3
Serviço #690 @ 7.89
07:31 B3
07:53 B4
08:14 B5
08:24 B6
Número do itinerário: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 